### DISCUSSION TITLE
Date naming conventions for simulations, YYYY0MMDD or YYYYMMDD

### DISCUSSION THREAD

**Discussion by oyvindseland** [NorESMhub/noresm3_dev_simulations#55] [Discussions about coupled simulations]
I have noticed that a couple of simulations gives the data as YYY0MMDD for October. Should we use a 0 in front of the month. 
If we want a zero between yest and month I think we should then use the same number of digits also for MM < 10

**Comment by gold2718** [comment]
I think this is a typo in the casename. It is not specified that way in the [issue](https://github.com/NorESMhub/noresm3_dev_simulations/issues/51).
Note that the zero is in the casename, not the date field of the filenames.
@monsieuralok ?

**Comment by monsieuralok** [comment]
@oyvindseland @gold2718 yes, it is extra zero is in CASE name by mistake. date field attached with filename should be correct.
